docs/development.md

```diff
@@ -1,418 +1,659 @@
+Waiting for deployment "cert-manager-cainjector" rollout to finish: 0 of 1 updated replicas are available...
+deployment "cert-manager-cainjector" successfully rolled out
+kubectl rollout status  deployment/cert-manager-webhook -n cert-manager
+deployment "cert-manager-webhook" successfully rolled out
+===> installing tyk-pro
 sh ./ci/deploy_tyk_pro.sh
 creating namespace tykpro-control-plane
+creating namespace tykpro-control-plane
 namespace/tykpro-control-plane created
+deploying gRPC plugin server
+service/grpc-plugin created
+deployment.apps/grpc-plugin created
+Waiting for deployment spec update to be observed...
+Waiting for deployment "grpc-plugin" rollout to finish: 0 out of 1 new replicas have been updated...
+Waiting for deployment "grpc-plugin" rollout to finish: 0 of 1 updated replicas are available...
+deployment "grpc-plugin" successfully rolled out
 deploying databases
 service/mongo created
 deployment.apps/mongo created
 deployment.apps/redis created
 service/redis created
 waiting for redis
-deployment.apps/redis condition met
+Waiting for deployment "redis" rollout to finish: 0 of 1 updated replicas are available...
+deployment "redis" successfully rolled out
 waiting for mongo
-deployment.apps/mongo condition met
+Waiting for deployment "mongo" rollout to finish: 0 of 1 updated replicas are available...
+deployment "mongo" successfully rolled out
 creating configmaps
 configmap/dash-conf created
 configmap/tyk-conf created
 setting dashboard secrets
 secret/dashboard created
-"OBFUSCATED"
-deploying dashboard
+ZXlKaGJHY2lPaUpTVXpJMU5pSXNJblI1Y0NJNklrcFhWQ0o5LmV5SmhiR3h2ZDJWa1gyNXZaR1Z6SWpvaU1USmlZek5tTkRJdFltSmxOeTAwWkRWaExUUXlORGd0TlRBMk1XTXhZV1V5TjJZd0xERmpPVFprWVRZMUxXVmhOVEF0TkdVM01TMDNNVFJtTFdFMlpUZ3dOVGcxWXpNMlpDd3pNR013TVdJM05DMWhPREF5TFRReU56TXRORGMzWlMxak0ySmtOREU0T0daaE5tSXNNRFl3TXpZd00yRXRNRGxtWVMwME1HRm1MVFkwWmpRdE56VTNabVE0TWpRNFkySTVMR0V5TVdKaE1UaGxMVGMwWVdRdE5HVXhZeTAwTkdJMkxXSmlNamswWXpVeU9EazBaU3hpWVRObU1UazJOaTB4TURWbUxUUmxOV010Tm1FM1pDMDBNV015TnpobU5UZzJOMlFzTkRkaU56RTJORGt0TkRBNVpDMDBNMkk1TFRZNVltTXROR0V3WmpVNVl6VTNOakl5TERVeE9UUXpOR0V6TFRZeE9XRXRORGRsWmkwMlpqZGpMVFUyTlRGaE1qZGtPV1kzTlN3M09UTTVNRGN5TkMxbE56TTFMVFJoTlRZdE5qWXhOQzFoWmpBMlpEYzVPV0ZrTXpNc1lqRmxOemN4WXpRdE1XRTJaUzAwWkdaa0xUWTFORGt0WXpKbFlUaGtOV0l6WWpJMElpd2laWGh3SWpveE5qQTROekEyTURjeExDSnBZWFFpT2pFMk1EWXhNVFF3TnpFc0ltOTNibVZ5SWpvaU5UYzNPVGN4TVRrME5XWTVNbVUyTmpnNU1EQXdNVEkzSWl3aWMyTnZjR1VpT2lKdGRXeDBhVjkwWldGdExISmlZV01zWjNKaGNHZ2lMQ0oySWpvaU1pSjkuZ2VsUC1YRmFqUVNxOGxGSUFvU0pfQWZLU1QwTm9MNnNEdUdMd056d1NSS1Z6bHJOUTFLWmFBU045UlAycjR4Mm5nNk1uYWhFdUZYamxOQW95Z1lxWENRdWpoYi1PWlVQWmMtVmZaYThYNVM3eTUtNGZMNi1ISUxnWlphczUxMEJxcjlsMXhobFh1WkM0WTA0TWdvVkRkVzBzV05mTUtEOWE3cXU1X3A4T3d5ZzRzODgtRmV5bE9hbWVaNnZCVXBmV2pZVnlzaUZIeEpRNkYzemoxV0ZjcFNkZWxVMU9GVHNmRUFjaENEeXh5Z0U3OTA4SGQ3eW5nb3ItZlg5UnVxdjFsLW9MS1VLZGJLYnhfQm9kaTRtQUdWbmVQQVVFTWlnbHZ4VkM5aFdRenNPQ1NtbEpQY05FX2c3U2k1Z2NCNkR6SGVQcFFsYzRTN1JqT3lEcTNIdUFndeploying dashboard
```

Comment:
Can we obfuscate the trial license key?

Reply:
we don't even need to show it here, done 7675c10